### ISSUE TITLE
CarPi

### ISSUE THREAD

**Issue by JoeBona1** [tigattack/CarPi#1]
Sorry if this is an ignorant question, is this a full version of OpenAuto with aasdk written in Pyton

**Comment by tigattack** [comment]
No, this is just a personal project which simply utilises OpenAuto, not a fork/rewrite of OpenAuto itself.
